hie-plugin-api/Haskell/Ide/Engine/PluginApi.hs

```diff
@@ -58,14 +58,23 @@ module Haskell.Ide.Engine.PluginApi
   , HIE.CachedInfo(..)
 
   -- * used for tests in HaRe
-  , HIE.BiosLogLevel(..)
-  , HIE.BiosOptions(..)
-  , HIE.defaultOptions
+  , BiosLogLevel(..)
+  , BiosOptions(..)
+  , defaultOptions
   ) where
 
+
+
 import qualified GhcProject.Types                    as GP
 import qualified Haskell.Ide.Engine.Ghc              as HIE
 import qualified Haskell.Ide.Engine.GhcModuleCache   as HIE (CachedInfo(..),HasGhcModuleCache(..))
 import qualified Haskell.Ide.Engine.ModuleCache      as HIE (ifCachedModule)
 import qualified Haskell.Ide.Engine.PluginsIdeMonads as HIE
 import qualified Language.Haskell.LSP.Types          as LSP ( filePathToUri )
+import qualified HIE.Bios.Types as HIE
+
+defaultOptions = HIE.defaultCradleOpts
+type BiosLogLevel = HIE.BIOSVerbosity
+
+type BiosOptions = HIE.CradleOpts
+runIdeGhcMBare = error "Not implemented"
```

Comment:
Not implemented, but also, not needed?

Reply:
I think HaRe needs this to run standalone.

Reply:
HaRe uses it in https://github.com/alanz/HaRe/blob/hie-plugin-api/src/Language/Haskell/Refact/Utils/Monad.hs#L224 
Which brings me to the idea: do we have to update HaRe to depend on the new plugin-api?